index.html

```diff
@@ -0,0 +1,33 @@
+<!-- Note: you may use this starter code, but it is not a requirement -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>The Gallery of Horns</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/base.css" />
+    <link rel="stylesheet" href="css/layout.css" />
```

Comment:
The file in your repo is actually named "layouts.css" so when we load your application, we cannot see the correct layout of the boxes side by side.

Please make sure and check your work in the browser before submitting so that you can catch these sorts of bugs.